content/en/docs/Getting Started/quickstart-kind.md

```diff
@@ -0,0 +1,168 @@
+---
+
+[Kind](https://kind.sigs.k8s.io/) is a tool for running Kubernetes locally using 
+Docker.
+
+## Prerequisites
+
+1.  [Install Docker](https://docs.docker.com/get-docker/)
+1.  [Install kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+1.  [Install kubectl](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+
+## Create your Kubernetes cluster
+
+Create a cluster named `tektoncd`:
+
+```bash
+kind create cluster --name tektoncd
+```
+
+The process takes a few seconds, you see the following output:
+
+<pre>
+Creating cluster "tektoncd" ...
+ ✓ Ensuring node image (kindest/node:v1.21.1) 🖼
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-tektoncd"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-tektoncd
+
+Have a nice day! 👋
+</pre>
+
+You can check that the cluster was successfully created with `kubectl`:
+
+```bash
+kubectl cluster-info --context kind-tektoncd
+```
+
+The output confirms that Kubernetes is running:
+
+<pre>
+Kubernetes control plane is running at https://127.0.0.1:39509
+CoreDNS is running at
+https://127.0.0.1:39509/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+</pre>
+
+## Install Tekton Pipelines
+
+1. To install the latest version of Tekton Pipelines, use `kubectl`:
+
+   ```bash
+   kubectl apply --context kind-tektoncd --filename \
+   https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+   ```
+
+1. Monitor the installation:
+
+   ```bash
+   kubectl get pods --context kind-tektoncd --namespace tekton-pipelines --watch
+   ```
+
+When all components show `Running` under the `STATUS` column the installation
+is complete.
+
+Hit *Ctrl + C* to stop monitoring.
+
+## Create and run a basic Task
+
+A **Taks** defines a series of **steps** that run sequentially to perform some
+build work. Every Task runs as a pod on your Kubernetes cluster, with each step
+running in its own container.
+
+1. To create a task, open your favorite editor and create a file named
+   `hello-world.yaml` with the following content:
+
+	 ```yaml
+	 apiVersion: tekton.dev/v1beta1
+	 kind: Task
+	 metadata:
+	 	name: echo-hello-world
+	 spec:
+	 	steps:
+	 		- name: echo
+	 			image: alpine
+	 			command:
+	 				- echo
+	 			args:
+	 				- "Hello World"
+	 ```
+
+1. Apply the Task to your cluster:
+
+   ```bash
+   kubectl apply --context kind-tektoncd --filename hello-world.yaml
+   ```
+
+1. To run this task, you must instantiate it using `TaskRun`. Create another
+   file named `hello-world-run.yaml` with the following content:
+
+   ```yaml
+   apiVersion: tekton.dev/v1beta1
+   kind: TaskRun
+   metadata:
+     name: echo-hello-world-task-run
+   spec:
+     taskRef:
+       name: echo-hello-world
+   ```
```

Comment:
probably makes sense to use TaskRuns explicitly but just so you know there's one other option that might be a nice user experience - but the user would have to also install `tkn` (https://github.com/tektoncd/cli) so maybe it's not worth the extra setup - but if it wasn't too much trouble, instead of making a TaskRun here the user could do [`tkn task start echo-hello-world`](https://github.com/tektoncd/cli/blob/main/docs/cmd/tkn_task_start.md)

Reply:
I thought about it, same to check the task status. The trade-off of using a simpler command but having to install something extra, if this is your first ever experience with tekton, may not be worth it in my opinion. For the longer tutorial covering pipelines in more detail it's definitely a good idea to ask for the `tkn` client prerequisite.